src/main/resources/locales/zh-CN.yml

```diff
@@ -1,1099 +1,1447 @@
-      name: 音符盒
+      description: "允许/禁止 使用音符盒"
+      name: "&a&l使用音符盒"
+      hint: "&c已被禁止使用音符盒"
+    OBSIDIAN_SCOOPING:
+      name: "&a&l黑曜石变熔岩"
+      description: |-
+        &7允许/禁止 玩家用桶把黑曜石变回熔岩
+        &7这可以帮助新手找回熔岩
+        &c注意： 如果黑曜石附近2格范围内还有
+        &c其它黑曜石，则不能把黑曜石变回熔岩
+      scooping: "&a已将黑曜石变回熔岩。 下次请小心！"
+      obsidian-nearby: "&c附近有其它黑曜石， 这个黑曜石不能变回熔岩。"
+    OFFLINE_GROWTH:
+      description: |-
+        &7允许/禁止 所有成员都已离线的岛屿上
+        &7的植物继续生长
+        &a这可以帮助减少性能开销
+      name: "&a&l离线生长"
     OFFLINE_REDSTONE:
       description: |-
-        &a当禁止时，所有成员离线的情况下
-        &a红石将不再动作。可能有助于减轻
-        &a卡顿情况。
-      name: 离线红石
+        &7允许/禁止 所有成员都已离线的岛屿上
+        &7的红石设备继续运行
+        &a这可以帮助减少性能开销
+        &a这个设置不会影响出生岛屿（主城）
+      name: "&a&l离线红石"
+    PETS_STAY_AT_HOME:
+      description: |-
+        &7允许/禁止 驯服的宠物始终待在自
+        &7己的岛上， 不会到别人的倒上去。
+      name: "&a&l宠物不离开岛屿"
     PISTON_PUSH:
-      description: '&a防止活塞将方块推出岛屿'
-      name: 活塞推动
+      description: "允许/禁止 活塞将方块推出岛屿范围"
+      name: "&a&l活塞推动保护"
     PLACE_BLOCKS:
-      description: 切换放置
-      hint: 禁止放置方块
-      name: 放置方块
-    PORTAL:
-      description: 切换使用
-      hint: 禁止使用传送门
-      name: 传送门
+      description: "允许/禁止 在岛上放置方块"
+      name: "&a&l放置方块"
+      hint: "&c已被禁止在岛上放置方块"
     POTION_THROWING:
-      hint: 不允许扔药水
-      name: 药水投掷
-      description: '&a切换允许/禁止投掷药水。包括喷溅药水和滞留药水。'
+      name: "&a&l投掷药水瓶"
+      description: |-
+        &7允许/禁止 在岛上投掷药水瓶
+        &7这包括喷溅型药水和滞留型药水
+      hint: "&c已被禁止投掷药水瓶"
+    NETHER_PORTAL:
+      description: "允许/禁止 使用下界传送门"
+      name: "&a&l使用下界传送门"
+      hint: "&c已被禁止使用下界传送门"
+    END_PORTAL:
+      description: "允许/禁止 使用末地传送门"
+      name: "&a&l使用末地传送门"
+      hint: "&c已被禁止使用末地传送门"
     PRESSURE_PLATE:
-      description: 切换使用
-      hint: 禁止使用压力板
-      name: 压力板
+      description: "允许/禁止 激活压力板"
+      name: "&a&l使用压力板"
+      hint: "&c已被禁止使用压力板"
     PVP_END:
-      description: '&c允许或禁止在末地 PVP。'
-      hint: 禁止在末地 PVP
-      name: 末地 PVP
+      description: "&c允许/禁止 在末地 PVP"
+      name: "&a&l末地 PVP"
+      hint: "&c已被禁止在末地 PVP"
+      enabled: "&e已允许在末地 PVP！"
+      disabled: "&a已禁止在末地 PVP。"
     PVP_NETHER:
-      description: '&c允许或禁止在下界 PVP。'
-      hint: 禁止在下界 PVP
-      name: 下界 PVP
+      description: "&c允许/禁止 在下界 PVP"
+      name: "&a&l下界 PVP"
+      hint: "&c已被禁止在下界 PVP"
+      enabled: "&e已允许在下界 PVP！"
+      disabled: "&a已禁止在下界 PVP。"
     PVP_OVERWORLD:
-      active: '&c这里允许 PVP！'
-      description: '&c允许或禁止岛上 PVP。'
-      hint: '&c禁止 PVP'
-      name: 主世界 PVP
+      description: "&c允许/禁止 在主世界 PVP"
+      name: "&a&l主世界 PVP"
+      hint: "&c已被禁止在主世界 PVP"
+      enabled: "&e已允许在主世界 PVP！"
+      disabled: "&a已禁止在主世界 PVP。"
     REDSTONE:
-      description: 切换使用
-      hint: 禁止使用红石物品
-      name: 红石物品
+      description: |-
+        &7允许/禁止 使用红石设备
+        &7包括红石线、中继器、比
+        &7较器和阳光感应器
+      name: "&a&l使用红石设备"
+      hint: "&c已被禁止使用红石设备"
+    # 英文原版只字未提末影龙，这让人很费解
+    # 代码里的功能确实是为了防止末影龙的生成
```

Comment:
Is this a bug that needs to be fixed?

Reply:
Yes, I think **en-US.yml** should be corrected.

Reply:
What is wrong or needs to be fixed? I can't see the problem right now.

Reply:
```
    REMOVE_END_EXIT_ISLAND:
      description: |-
        &a Prevent the ender dragon spawn
        &a in the end at coordinates 0,0.
      name: "Remove ender dragon"
```

Reply:
My English is not good. This is translated from Chinese.

Reply:
At first I thought it was to remove the island. . . . .
After reading the code, I knew that it was an ender dragon.